.ci/magician/cmd/generate_comment.go

```diff
@@ -338,38 +349,6 @@ func execGenerateComment(prNumber int, ghTokenMagicModules, buildId, buildStep,
 		errors["Other"] = append(errors["Other"], "Failed to update breaking-change status check with state: "+breakingState)
 	}
 
-	// Run missing test detector (currently only for beta)
-	missingTestsPath := mmLocalPath
-	for _, repo := range []source.Repo{tpgbRepo} {
-		if !repo.Cloned {
-			fmt.Println("Skipping missing tests; repo failed to clone: ", repo.Name)
-			continue
-		}
-		missingTests, err := detectMissingTests(missingTestsPath, repo.Path, oldBranch, rnr)
-		if err != nil {
-			fmt.Println("Error running missing test detector: ", err)
-			errors[repo.Title] = append(errors[repo.Title], "The missing test detector failed to run.")
-		}
-		data.MissingTests = missingTests
-	}
-
-	// Run unit tests for missing test detector (currently only for beta)
-	if pathChanged("tools/missing-test-detector", tpgbRepo.ChangedFiles) {
-		fmt.Printf("Found diffs in missing test detector:\n%s\nRunning tests.\n", diffs)
-		if err = runMissingTestUnitTests(
```

Comment:
Are we losing anything by _not_ running these tests on this branch? The diff-processor unit tests only run on the main branch of the downstreams. Are there any types of problems that this test run would have caught that will no longer be caught?

Reply:
Per discussion: the failures in ReadAll don't actually cause the _test_ to fail unless there's a panic. This matches the behavior of the missing test detector itself. So there is no case where a failure would previously have shown up which will no longer show up - and everything that would have been logged should still be getting logged.

However, we do need to (as part of this PR) make sure that the diff-processor unit tests are able to _run_ TestReadAllTests in a meaningful way (either by passing in SERVICES_DIR or by reworking TestReadAllTests to not require it, for example by using a relative dir to find the old / new directories locally.)